src/static/riot/competitions/tile/competition_tile.tag

```diff
@@ -63,15 +63,20 @@
             margin-bottom 35px !important
 
         .main-wrapper
-            border-left 5px solid lightgreen
+            border-left 5px solid #c7402d
```

Comment:
hmm, should we define these colors somewhere..............

Reply:
Added Color Reference to base_template.styl